### ISSUE TITLE
latissimus dorsi part-of arm?

### ISSUE THREAD

**Issue by cmungall** [obophenotype/mouse-anatomy-ontology#15]
The path here seems suspect:

```
      is_a EMAPA:16405 ! limb
       is_a EMAPA:17412 ! forelimb
        po EMAPA:17413 ! arm
         po EMAPA:17421 ! shoulder
          po EMAPA:36154 ! shoulder muscle 
           is_a EMAPA:18177 ! deltoid
           is_a EMAPA:18178 ! latissimus dorsi *** 
           is_a EMAPA:18181 ! subscapularis
           is_a EMAPA:18182 ! supraspinatus
           is_a EMAPA:18526 ! infraspinatus
           is_a EMAPA:36155 ! teres muscle
```

each individual relationship is justifiable given *some* definition of the child and parent, but I don't think there is any single consistent interpretation of the terms that justify the path. Perhaps we could say these muscles are functionally part of the arm. E.g. they attach to some arm bone. But for gene expression queries, querying for arm probably shouldn't return genes expressed near the iliac crest...

**Comment by cmungall** [comment]
In uberon, in discussion with the wider phenoscape plus MA/ZFA group, we decided on distinguishing (free) limbs from pectoral/pelvic complexes. We keep the shoulder in the latter. The label "arm" is a bit fuzzy but we decided to make it the sum of stylopod and zeugopod, and in uberon this connects to the shoulder. "shoulder muscle" is defined as a muscle that is part of shoulder, but this is probably wrong, except for a very broad def of shoulder. Should probably be defined by attachment to a shoulder bone.
